neonvm/tools/vm-builder-generic/main.go

```diff
@@ -298,98 +308,40 @@ func main() {
 		log.Fatalln(err)
 	}
 
-	var SrcImageSpec ImageSpec
+	tmplCtx := TemplatesContext{
+		SrcImage:   *srcImage,
+		Entrypoint: imageSpec.Config.Entrypoint,
+		Cmd:        imageSpec.Config.Cmd,
+		Env:        imageSpec.Config.Env,
+	}
 
 	if len(imageSpec.Config.User) != 0 {
```

Comment:
if this is a string let's do != ""

Reply:
ehhh, yes 100% agree. There's kind of a lot going on in this PR already; I'll open another PR to do this (and others) separately.